src/spikeinterface/extractors/extractorlist.py

```diff
@@ -29,7 +29,7 @@
 from .nwbextractors import NwbRecordingExtractor, NwbSortingExtractor, read_nwb, read_nwb_recording, read_nwb_sorting
 
 from .cbin_ibl import CompressedBinaryIblExtractor, read_cbin_ibl
-from .iblstreamingrecording import IblStreamingRecordingExtractor, read_ibl_streaming_recording
+from .iblextractors import IblRecordingExtractor, IblSortingExtractor, read_ibl_recording, read_ibl_sorting
```

Comment:
Maybe we should keep the old with a deprecation warning?

Reply:
I think it was only used internally and for testing. Fine to remove :)